draft-ietf-quic-multipath.md

```diff
@@ -995,24 +996,24 @@ sequence number value.
 
 The Retire Prior To field indicates which connection IDs
 should be retired among those that share the Path ID in the Path Identifier field.
-Connection IDs associated with different path IDs are not affected.
+Connection IDs associated with different Path IDs are not affected.
 
 Note that the NEW_CONNECTION_ID frame can only be used to issue or retire
 connection IDs for the initial path with Path ID 0.
 
```

Comment:
```suggestion
```
Proposal in the other PR was to remove this sentence as this is already covered in the path management section.